includes/post-types.php

```diff
@@ -76,7 +76,7 @@ function give_setup_post_types() {
 
 	$give_forms_args = array(
 		'labels'             => $give_forms_labels,
-		'public'             => true,
+		'public'             => $give_forms_singular,
 		'publicly_queryable' => $give_forms_singular,
```

Comment:
@mehul0810 do we need this param, it will set automatically to `false` if `public` is `false` and vice Versa, right?